src/shared/components/cancerSummary/CancerSummaryContent.tsx

```diff
@@ -124,9 +142,8 @@ export class CancerSummaryContent extends React.Component<ICancerSummaryContentP
                     });
                     return memo;
                 }, [] as IBarGraphDataset[]);
+                const orderedAlts = _.keys(orderedLabels);
```

Comment:
is this definitely stable? keys usually isnt order-guaranteed. It would probably be better to make orderedLabels an array of objects `{ alteration: keyof ICancerTypeAlterationCounts, label: string }`